joomla/joomla-cms
# [4.0] Remove ext/mysql database driver

### Summary of Changes

Following https://github.com/joomla/joomla-cms/pull/14136 and the related RFC, this PR removes the MySQL database driver supporting the deprecated PHP `ext/mysql`.

### Testing Instructions

Note that `JDatabaseDriverMysql` and its supporting elements have been removed.

On a server where the configuration is set to `public $dbtype = 'mysql';` `JDatabaseDriver::getInstance()` should try to establish a database connection using `ext/mysqli` first then `ext/pdo`.  If neither of those available, the database connection is not established.

### Documentation Changes Required

Update https://docs.joomla.org/Potential_backward_compatibility_issues_in_Joomla_4 with the removal information.
